src/pybert/gui/view.py

```diff
@@ -55,7 +52,7 @@ class MyHandler(Handler):
     def do_run_simulation(self, info):
         """Spawn a simulation thread and run with the current settings."""
         the_pybert = info.object
-        if self.run_sim_thread and self.run_sim_thread.isAlive():
+        if self.run_sim_thread and self.run_sim_thread.is_alive():
```

Comment:
Is this just a name change, or are you pointing to a different function?
(I couldn't find an equivalent change to the `isAlive()` function definition; is that ours or in a library?)

Reply:
Added in 2.6 just to be pep8 or underscore vs. camelCase.

- https://docs.python.org/2.7/library/threading.html#threading.Thread.isAlive

Removed in 3.9

- https://docs.python.org/3/whatsnew/3.9.html